container/podman/podman.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package podman
+
+import (
+	"fmt"
+
+	v1 "github.com/google/cadvisor/info/v1"
+	"github.com/google/cadvisor/machine"
+
+	"github.com/containers/podman/v3/pkg/bindings/system"
+)
+
+func Status() (v1.PodmanStatus, error) {
+	client, err := Client()
+	if err != nil {
+		return v1.PodmanStatus{}, fmt.Errorf("unable to commmunicate with podman daemon: %v", err)
+	}
+
+	podmanInfo, err := system.Info(client, nil)
+	if err != nil {
+		return v1.PodmanStatus{}, err
+	}
+
+	status := v1.PodmanStatus{}
+	status.KernelVersion = machine.KernelVersion() // TODO (Creatone): Check if it should came from Podman.
```

Comment:
Podman does not know anything about kernel.

Reply:
Are you sure? Running `podman system info` gives Kernel version.
Also, there is the code that checks that: 
https://github.com/containers/podman/blob/eeb71490e57047202410821fe98fa4332371cb03/libpod/info.go#L330-L340

Reply:
Sure, but it is just getting the kernel off of the host machine. I guess the podman-remote system info does get the kernel on the remote machine.